packages/components/Link/src/Link.tsx

```diff
@@ -72,7 +72,7 @@ export const Link = compose<ILinkType>({
 
     // create the merged slot props
     const slotProps = mergeSettings<ILinkSlotProps>(styleProps, {
-      root: { ...linkProps, ref: linkRef },
+      root: { ...linkProps, ref: linkRef, onAccessibilityTap: onAccTap },
```

Comment:
>onAccessibilityTap [](start = 42, length = 18)

it would be good to have automated tests that check this. What is automation story for these controls?